### DISCUSSION TITLE
Field descriptions within arrays should be aware of its index

### DISCUSSION THREAD

**Discussion by tyteen4a03** [payloadcms/payload#4360] [Feature Requests & Ideas]
It would be great if custom React components (e.g. for admin.description) used by fields within arrays can be made aware of the index of the array entry.

**Comment by JarrodMFlesch** [comment]
The description components could just take a new prop for "path" and then you would be able to determine the index from the pathname passed through. Would you like to take a crack at a PR? Would be a pretty simple one and I could lead you in the right direction some time this week.

**Reply by tyteen4a03** [reply]
Made a quick PR: https://github.com/payloadcms/payload/pull/4364

**Reply by JarrodMFlesch** [reply]
Thanks! Looks great, I'll check it out on my laptop in the morning 👍
